docs/notebooks-installation.md

```diff
@@ -43,20 +43,18 @@ The table below lists the supported operating systems and Python versions.
 |                                     | (64-bit                        |
 |                                     | ) <https://www.python.org/>`__ |
 +=====================================+================================+
-| Ubuntu 18.04 LTS                    | 3.6, 3.7, 3.8, 3.9, 3.10       |
+| Ubuntu 18.04 LTS                    | 3.7, 3.8, 3.9, 3.10            |
 +-------------------------------------+--------------------------------+
-| Ubuntu 20.04 LTS                    | 3.6, 3.7, 3.8, 3.9, 3.10       |
+| Ubuntu 20.04 LTS                    | 3.7, 3.8, 3.9, 3.10            |
 +-------------------------------------+--------------------------------+
-| Red Hat Enterprise Linux 8          | 3.6, 3.8, 3.9, 3.10            |
+| Red Hat Enterprise Linux 8          | 3.8, 3.9, 3.10                 |
 +-------------------------------------+--------------------------------+
-| CentOS 7                            | 3.6, 3.7, 3.8, 3.9, 3.10       |
```

Comment:
Did we agree to remove information about CentOS?

Reply:
because we don't officially support it since 2022.1